Ryujinx.HLE/HOS/Kernel/SupervisorCall/Syscall.cs

```diff
@@ -2107,6 +2107,34 @@ public KernelResult GetInfo(out ulong value, InfoType id, int handle, long subId
                         break;
                     }
 
+                case InfoType.MesosphereCurrentProcess:
+                    {
+                        if (handle != 0)
+                        {
+                            return KernelResult.InvalidHandle;
+                        }
+
+                        if ((ulong)subId != 0)
+                        {
+                            return KernelResult.InvalidCombination;
+                        }
+
+                        KProcess currentProcess = KernelStatic.GetCurrentProcess();
+                        KHandleTable handleTable = currentProcess.HandleTable;
+                        int outHandle = 0;
+
+                        KernelResult result = handleTable.GenerateHandle(currentProcess, out outHandle);
```

Comment:
```suggestion

                        KernelResult result = handleTable.GenerateHandle(currentProcess, out int outHandle);
```
Could you define `outHandle` inline?